coral-common/src/main/java/com/linkedin/coral/common/transformers/SqlCallTransformers.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.common.transformers;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.calcite.sql.SqlCall;
+
+
+/**
+ * Container for SqlCallTransformer
+ */
+public class SqlCallTransformers {
+  private final List<SqlCallTransformer> operatorTransformers;
+
+  SqlCallTransformers(List<SqlCallTransformer> operatorTransformers) {
+    this.operatorTransformers = operatorTransformers;
+  }
+
+  public static SqlCallTransformers of(SqlCallTransformer... operatorTransformers) {
+    return new SqlCallTransformers(Arrays.asList(operatorTransformers));
+  }
```

Comment:
Usually `of` is associated with immutable collections. Not sure it is reasonable to have both `add` and `of`. Something to consider.